elasticsearch/src/main/java/com/amazon/opendistroforelasticsearch/sql/elasticsearch/storage/ElasticsearchIndexScan.java

```diff
@@ -83,4 +85,11 @@ public ExprValue next() {
         return ExprValueUtils.fromObjectValue(hits.next().getSourceAsMap());
```

Comment:
Not a problem of this pr, but sometimes the result of method `getSourceAsMap()` is not what we expected, e.g. the source would be all fields when the designated field(s) is an evaluation function (like `abs(balance)`). Here is a similar issue: https://github.com/opendistro-for-elasticsearch/sql/issues/461. Are we likely running into the same problem here?

Reply:
Yeah, you're right, this is only for the time being. Need to handle different cases in future.